src/test/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringUtilTest.kt

```diff
@@ -1,129 +1,215 @@
+    }
+
+    @Test
+    fun `Test 'nytt beløp', 'tidligere utbetalt' og 'resultat' for simuleringsperiode med reduksjon i feilutbetaling`() {
+        val posteringer =
+                posteringer(juli(2021), beløp = 100, posteringstype = YTELSE) +
+                posteringer(juli(2021), beløp = 100, posteringstype = YTELSE) +
+                posteringer(juli(2021), beløp = -99, posteringstype = YTELSE) +
+                posteringer(juli(2021), beløp = -99, posteringstype = YTELSE) +
+                posteringer(juli(2021), beløp = 98, posteringstype = FEILUTBETALING) +
+                posteringer(juli(2021), beløp = -99, posteringstype = FEILUTBETALING)
+
+        val simuleringsperioder = grupperPosteringerEtterDato(
+                posteringer.tilSimuleringMottakere())
+
+        Assertions.assertEquals(1, simuleringsperioder.size)
+
+        Assertions.assertEquals(BigDecimal.valueOf(200), simuleringsperioder[0].nyttBeløp)
+        Assertions.assertEquals(BigDecimal.valueOf(197), simuleringsperioder[0].tidligereUtbetalt)
+        Assertions.assertEquals(BigDecimal.valueOf(1), simuleringsperioder[0].resultat)
+    }
+
+    val simulertePosteringerMedNegativFeilutbetaling =
+            posteringer(juli(2021), beløp = -500, posteringstype = FEILUTBETALING) +
+            posteringer(juli(2021), beløp = -2000, posteringstype = YTELSE) +
+            posteringer(juli(2021), beløp = 3000, posteringstype = YTELSE) +
+            posteringer(juli(2021), beløp = -500, posteringstype = YTELSE)
+
+    @Test
+    fun `Total etterbetaling skal bli summen av ytelsene i periode med negativ feilutbetaling`() {
+        val restSimulering = lagSimuleringsoppsummering(
+                simulertePosteringerMedNegativFeilutbetaling.tilDetaljertSimuleringsresultat(),
+                15.august(2021)
         )
-        assertThat(posteringerGruppert.first().nyttBeløp).isEqualTo(beløp)
-        assertThat(posteringerGruppert.last().nyttBeløp).isEqualTo(nyttBeløp)
-        assertThat(posteringerGruppert.first().fom).isEqualTo(fraDato)
-        assertThat(posteringerGruppert.last().fom).isEqualTo(fraDato.plusMonths(antallMåneder.toLong()))
-        assertThat(simuleringsresultatDto.etterbetaling).isEqualTo(beløp.multiply(BigDecimal(antallMånederEtterStart - antallMånederFeilutbetalt)))
-        assertThat(simuleringsresultatDto.feilutbetaling).isEqualTo(totaltFeilutbetaltBeløp)
-        assertThat(simuleringsresultatDto.fom).isEqualTo(fraDato)
-        assertThat(simuleringsresultatDto.forfallsdatoNestePeriode).isEqualTo(tidSimuleringHentet.with(TemporalAdjusters.lastDayOfMonth()))
+
+        Assertions.assertEquals(BigDecimal.valueOf(500), restSimulering.etterbetaling)
+    }
+
+    @Test
+    fun `Total feilutbetaling skal bli 0 i periode med negativ feilutbetaling`() {
+        val restSimulering = lagSimuleringsoppsummering(
+                simulertePosteringerMedNegativFeilutbetaling.tilDetaljertSimuleringsresultat(),
```

Comment:
Antar at noen av testene er reelle mtp det som er meldt inn som "feil" - hvis ikke er det sikkert nyttig å få Øystein til å sette opp noen testcaser slik at vi har dekning for corner-casene han snakket om :)